adapters/rubicon/rubicon.go

```diff
@@ -919,6 +921,9 @@ func resolveBidFloorAttributes(bidFloor float64, bidFloorCur string) (float64, s
 
 func updateExtWithIabAttribute(target json.RawMessage, data []openrtb2.Data, segTaxes []int) (json.RawMessage, error) {
 	var segmentIdsToCopy = getSegmentIdsToCopy(data, segTaxes)
+	if len(segmentIdsToCopy) == 0 {
+		return target, nil
+	}
```

Comment:
The test case for an empty or missing data array looked odd, because the `iab` target element is still created, but is empty. Please advise on if this aspect should be removed.